components/forms/w-toggle.vue

```diff
@@ -1,59 +1,58 @@
+        <w-toggle-item
+          v-model="model"
+          :type="type"
+          :radioButton="radioButton"
+          :disabled="disabled"
+          :invalid="invalid"
+          :equalWidth="equalWidth"
+          :name="id + ':toggles'"
+          :key="id + i + type"
+          v-bind="toggle"
+          @blur="triggerValidation" />
+      </div>
     </div>
   </w-field>
 </template>
 
-<script>
-import { computed } from 'vue'
-import { default as wField, fieldProps } from './w-field.vue'
-import { wToggleItem } from '#generics'
-import { createModel } from 'create-v-model'
+<script setup>
+import { computed } from 'vue';
+import { toggle as ccToggle } from '@warp-ds/component-classes';
+import { default as wField, fieldProps } from './w-field.vue';
+import { wToggleItem } from '#generics';
+import { createModel } from 'create-v-model';
 
-const hasLabelAndValue = e => ('value' in e) && ('label' in e)
+const props = defineProps({
+  ...fieldProps,
+  radio: Boolean,
+  checkbox: Boolean,
+  radioButton: Boolean,
+  equalWidth: Boolean,
+  small: Boolean,
+  disabled: Boolean,
+  invalid: Boolean,
+  toggles: {
+    type: Array,
+    required: true,
+    validator: (v) => v.every(hasLabelAndValue)
+  }
+});
+const emit = defineEmits(['update:modelValue']);
+const model = createModel({ props, emit });
+const type = computed(() => (props.radio || props.radioButton) ? 'radio' : 'checkbox');
+const role = computed(() => props.toggles.length > 1 ? ((props.radio || props.radioButton) ? 'radiogroup' : 'group') : undefined);
+const wrapperClasses = computed(() => ({
+  [ccToggle.wrapper]: true,
+  [`${ccToggle.segmentControl} ${ccToggle.focusableWithin}`]: props.radioButton,
+  [ccToggle.scJustified]: props.equalWidth,
+}));
 
-export default {
-  name: 'wToggle',
-  components: { wField, wToggleItem },
-  inheritAttrs: false,
-  props: {
-    ...fieldProps,
-    radio: Boolean,
-    checkbox: Boolean,
-    radioButton: Boolean,
-    equalWidth: Boolean,
-    small: Boolean,
-    disabled: Boolean,
-    toggles: {
-      type: Array,
-      required: true,
-      validator: (v) => v.every(hasLabelAndValue)
-    }
-  },
-  setup: (props, { emit }) => ({
-    model: createModel({ props, emit }),
-    type: computed(() => (props.radio || props.radioButton) ? 'radio' : 'checkbox'),
-    role: computed(() => props.toggles.length > 1 ? ((props.radio || props.radioButton) ? 'radiogroup' : 'group') : undefined),
-    wrapperClasses: computed(() => ({
-      'input-toggle--is-disabled': props.disabled && !props.radioButton,
-      'segment-control': props.radioButton,
-      'segment-control--justified': props.equalWidth,
-      'segment-control--small': props.small,
-      'segment-control--is-disabled': props.disabled && props.radioButton,
-      'input-toggle': props.radio || props.checkbox
-    })),
-    optionsClasses: computed(() => ({
-      'flex flex-row segment-control-options': props.radioButton,
-    }))
```

Comment:
Couldn't fins any usages of these so it is now removed but please enlighten me if I've missed something